woocommerce/google-listings-and-ads
# Multi-country targeting campaign - Integrate with campaign fetching and creation APIs

### Changes proposed in this Pull Request:

This PR implemented the rest of "**Integrate new API design with finished implementations**" in #1248.

- Revert all changes in https://github.com/woocommerce/google-listings-and-ads/commit/1b92a8522ad4d3e8d7805ad0cc26658663a773be

💡  The reverted mocking code has no side effects, so the code review could be able to take a quick look or skip it. 😎 

### Screenshots:

https://user-images.githubusercontent.com/17420811/158923334-f6e6ae0a-e18b-482d-b505-ef3cd9bd7702.mp4

### Detailed test instructions:

1. Go to create a campaign.
1. Select multiple targeting countries and finish the creation.
1. After navigating back to the Dashboard page, check the list below to see if there are multiple countries for the campaign just created.
1. Go to that campaign's edit page.
1. Check if the countries in the disabled selector are the same as when creating.

### Changelog entry
